src/analytics/lambdas/custom-resource/create-schemas.ts

```diff
@@ -262,9 +262,12 @@ function getCreateOrUpdateSchemasSQL(newAddedAppIdList: string[], props: Resourc
       database_name: props.projectId,
       schema: app,
       table_event: odsTableNames.event,
-      table_event_parameter: odsTableNames.event_parameter,
```

Comment:
You need to remove all legacy tables, event, user, and item.

Reply:
Remove all